### PR TITLE
Rendergraphbutton

### PR DESCRIPTION
**Pull request by Marc-Week**
Adds sumbit button click handlers (mouse and enter) and id references to the Hierarchy view. Merge after hierarchy-endpoint

### CODE REVIEW

**Comment by kathtran** [comment]
@Marc-Week You should be merging into our `development` branch, **NOT** `master`

File: src/hierarchy.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  document.getElementById('itemHierarchy').addEventListener('click', (event) => {
+    if (event.target.nodeName === 'LI') {
+      selected = event.target.id;
+    }
+  });
+  document.getElementById('renderButton').addEventListener('click', (event) => {
+    // TODO call render_Graph with selected as argument
+    console.log(selected);
+  });
+  document.getElementById('itemHierarchy').onkeydown = (event) => {
```
Comment (RickyV33) [inline src/hierarchy.js]: Add an event listener instead. 
`document.getElementById('itemHierarchy').addEventListener('keydown', event => {..}...`
Also, for single arguments passed into arrow functions, you don't need to close them in parenthesis

File: src/hierarchy.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    if (event.keyCode === 13) {
```
Comment (RickyV33) [inline src/hierarchy.js]: Add a comment saying what this does.

**Comment by Marc-Week** [comment]
Should i remove this PR and start a new one to merge into Dev?

**Pull request closed without merging**